cost/budget_report_alerts/budget_report_alerts.pt

```diff
@@ -8,7 +8,7 @@ tenancy "single"
 default_frequency "daily"
 
 info(
-  version: "2.1",
+  version: "2.2",
```

Comment:
Since we're adding a parameter, this should be a major version change to 3.0 instead of a minor version change to 2.2.